packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -158,6 +158,32 @@ function formatSampleBrowserFyi(violations: Violation[]): string {
   return formatExpandable(summary, detail);
 }
 
+/**
+ * It formats an array of Violation for informing removal of frozen region tags.
+ */
+function formatFrozenRegionTagFyi(violations: Violation[]): string {
+  let summary = 'You are about to delete the following frozen region tag';
```

Comment:
I would be tempted to pull this into a `util.ts`, so that you can test it in isolation (without all the setup required using nock).

Reply:
Done